src/registries/menu_items_registry.ts

```diff
@@ -9,7 +9,7 @@ export class MenuItemRegistry extends Registry<ActionSpec> {
   /**
    * @override
    */
-  add(key: string, value: ActionSpec): MenuItemRegistry {
+  replace(key: string, value: ActionSpec): this {
     if (value.id === undefined) {
       value.id = key;
     }
```

Comment:
you should also do the same checks for `addChild` no ? Or do we not care about them ?

Reply:
for consistency, I think we could remove `options.force` and introduce `replaceChild`